New Workflow - Sync Blacklisted URLs across Zscaler and Palo Alto Firewall (#421)
rapid7/insightconnect-workflows: workflows/Sync_Blacklisted_URLs_Across_Zscaler_and_Palo_Alto_Firewall/help.md

# Description

> The workflow automatically synchronizes URL blacklists across Palo Alto Firewall and Zscaler by adding or removing address objects from an address object group in your Palo Alto Firewall and blacklisting or unblacklisting URLs in Zscaler on an hourly schedule.

Reviewer: ```suggestion
The workflow automatically synchronizes URL blacklists across Palo Alto Firewall and Zscaler by adding or removing address objects from an address object group in your Palo Alto Firewall and blacklisting or unblacklisting URLs in Zscaler to keep them in sync on an hourly schedule.
```

# Key Features

> * **Sync URL blacklists** - Use InsightConnect to synchronize blacklisted URLs across Palo Alto Firewall and Zscaler easy.

Reviewer: ```suggestion
* **Sync URL blacklists** - Use InsightConnect to synchronize blacklisted URLs across Palo Alto Firewall and Zscaler easily.
```

# Description

The workflow automatically synchronizes URL blacklists across Palo Alto Firewall and Zscaler by adding or removing address objects from an address object group in your Palo Alto Firewall and blacklisting or unblacklisting URLs in Zscaler to keep them in sync bi-directionally on an hourly schedule.

> This workflow ensures that hosts blocked in the firewall are also blocked at the web proxy, and that hosts blocked in the web proxy are also blocked in the firewall creating a more consistent defensive perimeter across the organization.

Reviewer: ```suggestion
This workflow ensures that hosts blocked in the firewall are also blocked at the web proxy, and that hosts blocked in the web proxy are also blocked at the firewall, creating a more consistent defensive perimeter across the organization.
```

# Key Features

* **Sync URL blacklists** - Use InsightConnect to synchronize blacklisted URLs across Palo Alto Firewall and Zscaler easy.

# Requirements

* Palo Alto Firewall username and password
* Zscaler base URL, API key, username and password

# Documentation

## Setup

Import the workflow from the Rapid7 Extension Library and proceed through the Import Workflow wizard in InsightConnect. Import plugins, create or select connections, and rename the workflow as a part of the Import Workflow wizard as necessary.

This workflow leverages InsightConnect's Parameters feature. This feature allows variables used multiple times throughout a workflow to be entered once and then referenced throughout the workflow. 

> There are one parameter you will need to configure in order to complete setup of your workflow:

Reviewer: ```suggestion
There is one parameter you will need to configure in order to complete the setup of your workflow:
```

> * `Group Name`: The Palo Alto Firewall address object group to monitor

Reviewer: ```suggestion
* `Group Name`: The Palo Alto Firewall address object group to monitor. This is the group from which address objects will be synced with Zscaler.
```
